service/isiService.go

```diff
@@ -464,9 +463,73 @@ func (svc *isiService) OtherClientsAlreadyAdded(ctx context.Context, exportID in
 	return clientFieldsNotEmpty && !isNodeInClientFields && !isNodeFQDNInClientFields
 }
 
-func (svc *isiService) AddExportClientNetworkIdentifierByIDWithZone(ctx context.Context, exportID int, accessZone, nodeID string, addClientFunc func(ctx context.Context, exportID int, accessZone, clientIP string) error) error {
+// updateClusterToNodeIDMap updates cluster to nodeID map from input clusterName, nodeID and clientToUse
+func updateClusterToNodeIDMap(ctx context.Context, clusterToNodeIDMap *sync.Map, clusterName, nodeID, clientToUse string) error {
```

Comment:
our files are getting long, since these are not related to service calls, we can have a different file for the newly added functions.

Reply:
though these are not CSI RPC calls, these are helper functions. These are not even util functions that can be moved to utils package- these functions work on sync map, and it is not a good idea to have functions operate on this from another package.
Agree that we can have a new file for service related helper functions/methods, we can take up that as part of a separate ticket. There are many more such functions present in service package.